src/compile/mark/encode/base.ts

```diff
@@ -45,27 +39,7 @@ export function baseEncodeEntry(model: UnitModel, ignore: Ignore) {
   };
 }
 
-// TODO: mark VgValueRef[] as readonly after https://github.com/vega/vega/pull/1987
-function wrapAllFieldsInvalid(model: UnitModel, channel: Channel, valueRef: VgValueRef | VgValueRef[]): VgEncodeEntry {
```

Comment:
FYI, all old code in this file were for the legacy **secret/hidden** "hide" mode that I remove. 

The old hide mode hides data by adjusting fill/stroke, which still wastes rendering time to draw transparent marks. 
With the new `break-paths-keep-domain`, we're switching to rely on filters to produce this behavior.